app/webpacker/scripts/autocomplete.spec.js

```diff
@@ -1,4 +1,4 @@
-import initAutocomplete, { request } from "./autocomplete";
+import {getPath, initAutocomplete, request} from "./autocomplete";
```

Comment:
`initAutocomplete` is a default export 

i.e.

```
import initAutocomplete, {getPath, request} from "./autocomplete";
```

Reply:
changed, thanks!